### DISCUSSION TITLE
Export-FalconConfig running as Task in FlightControl for multiple tenants, data lose

### DISCUSSION THREAD

**Discussion by D3nCyb3rSec** [CrowdStrike/psfalcon#329] [Q&A]
I have a problem with Export-FalconConfig running as task for multiple tenants with one API in the master. Some exports are complete and some are not complete. If make this manual or for only one tenant it is working. For example some json files are missing in the ZIP file. I am grateful for any help.

**Comment by bk-cs** [comment]
`Export-FalconConfig` will only export if the results are available. Is it possible that you are being rate limited, or that the child CID does not have the licensing/configured items that would produce the json file?

**Reply by D3nCyb3rSec** [reply]
As i wrote if i make a manual / separate export of one CID it is working i get all the data. We have license and items are configured. I can see the configuration in the json file. It only works not if i try it in a task scheduler with more CIDs.

**Reply by bk-cs** [reply]
Does it work for those CIDs when you do them individually?

**Reply by D3nCyb3rSec** [reply]
Yes
